provider/equinixmetal/vendor/vendor.json

```diff
@@ -0,0 +1,13 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "kItpnJu5G9MVC56waIyHLoQZt/s=",
```

Comment:
I don't know how to generate `checksumSHA1`. Any pointers?

Reply:
Is there a way to automate the creation of these vendors/ directories, or just the vendor.json file?